de.metas.business/src/main/java/org/adempiere/impexp/product/MProductImportTableSqlUpdater.java

```diff
@@ -534,4 +534,15 @@ private void dbUpdateErrorMessages(@NonNull final String whereClause)
 		no = DB.executeUpdateEx(sql.toString(), ITrx.TRXNAME_ThreadInherited);
 		logger.warn("No Mandatory Pharma product category Name ={}", no);
 	}
+	
+	public void dbUpdateIsPriceCreated(@NonNull final String whereClause)
+	{
+		StringBuilder sql;
+		sql = new StringBuilder("UPDATE ")
+				.append(targetTableName + " i ")
+				.append(" SET IsPriceCreated = 'Y' ")
```

Comment:
please use the constant instead of the hardcoded string. otherwise it's hard to find the use of this physical column in our code